prov/verbs/src/fi_verbs.c

```diff
@@ -614,6 +615,11 @@ fi_ibv_getepinfo(const char *node, const char *service,
 	ret = rdma_create_ep(id, rai, NULL, NULL);
 	if (ret) {
 		ret = -errno;
+		if (ret == -ENOENT) {
+			FI_LOG(1, "verbs",
+				"rdma_create_ep() returned ENOENT.  We are likely dealing with a usnic_X device that has rdma_cm bugs, so skip the verbs provider instead.\n");
```

Comment:
I don't adhere strictly to the 80 character limit, but I do try to stop close to there (90-95 characters).  I do a lot of development on smaller terminal windows.  Please either shorten the text or split it across multiple lines